editoast/src/map/mod.rs

```diff
@@ -11,47 +11,47 @@ pub use self::layer_cache::get_layer_cache_prefix;
 pub use self::layer_cache::get_view_cache_prefix;
 pub use self::layer_cache::Tile;
 use crate::error::Result;
-use crate::RedisConnection;
+use crate::ValkeyConnection;
 
 /// Invalidates layer cache for a specific infra and view if provided
 ///
 /// # Arguments
 ///
-/// * `redis_pool` - Pool to use to connect to the redis
+/// * `valkey` - Pool to use to connect to the valkey
 /// * `infra_id` - Infra on which the layer must be invalidated
 /// * `layer_name` - Layer to invalidate
 ///
 /// Returns the number of deleted keys
 async fn invalidate_full_layer_cache(
-    redis: &mut RedisConnection,
+    valkey: &mut ValkeyConnection,
     infra_id: i64,
     layer_name: &str,
 ) -> Result<u64> {
     let prefix: String = get_layer_cache_prefix(layer_name, infra_id);
-    let matching_keys: Vec<String> = redis.keys(format!("{prefix}.*")).await?;
+    let matching_keys: Vec<String> = valkey.keys(format!("{prefix}.*")).await?;
     if matching_keys.is_empty() {
         return Ok(0);
     }
-    let number_of_deleted_keys = redis.del(matching_keys).await?;
+    let number_of_deleted_keys = valkey.del(matching_keys).await?;
     Ok(number_of_deleted_keys)
 }
 
 /// Invalidates all map layers of a specific infra
 ///
 /// # Arguments
 ///
-/// * `redis_pool` - Pool to use to connect to the redis
-/// * `layers` - Layers to invalidate
+/// * `valkey` - Pool to use to connect to the valkey
+/// * `layers` - Layers to valkey
```

Comment:
Shouldn't this stay `invalidate`? Why the change?

```suggestion
/// * `layers` - Layers to invalidate
```